src/modules/python/pbs_hooks/PBS_alps_inventory_check.PY

```diff
@@ -155,11 +158,12 @@ def get_apstat_nids(msg):
     # not ship the subprocess module
     apstat_out = os.popen(cmd_apstat)
 
+    pattern = re.compile("(.*?)\s+(.*?)\s+(.*?)\s+(.*?)\s+(.*)")
+

```

Comment:
That's interesting and good to know to use the ? in that fashion.  I'm not quite sure I understand why it should work, but that's OK.  Most of my regex training was 20 years ago when regex was much simpler.